### DISCUSSION TITLE
celsius symbols not showing in tmux

### DISCUSSION THREAD

**Discussion by jothi-prasath** [aristocratos/btop#579] [General]
btop without tmux:
![image](https://github.com/aristocratos/btop/assets/64248628/0c10f435-dcea-4009-8df2-3b66fc0271c2)

btop in tmux:
![image](https://github.com/aristocratos/btop/assets/64248628/5b7a7473-e499-40cd-b425-4726b1426c40)

the celsius symbols and graph symbols are not displaying properly in tmux

**Comment by deckstose** [comment]
Please file a bug report

**Comment by jothi-prasath** [comment]
#580
